src/js/node/http.ts

```diff
@@ -591,8 +677,12 @@ Server.prototype = {
             return pendingResponse;
           }
 
+          server[connectionsSymbol]++;
           var { promise, resolve: resolveFunction, reject: rejectFunction } = $newPromiseCapability(GlobalPromise);
-          return promise;
+          return promise.finally(() => {
```

Comment:
This adds an extra microtask tick to every node:http response.

Is that necessary?